### ISSUE TITLE
DB: Setting up a new Cluster/DBase

### ISSUE THREAD

**Issue by paulywill** [freecodecampBarrie/rubberduck#1]
Need a place to put the data!

![image](https://user-images.githubusercontent.com/602422/119865197-6bca2780-bee9-11eb-8d25-66b759dc0673.png)

**Comment by paulywill** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Setup MERN stack](https://trello.com/c/KoewvfFM/1-setup-mern-stack)

**Comment by paulywill** [comment]
Tried creating a fresh Cluster using the freeCodeCampBarrie gmail address, but kept getting an error.

`MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/`

Decided to use an existing database that was working.

I will continue to troubleshoot what I messed up.

**Comment by paulywill** [comment]
I think the issue was my machine.

I am using a USB Live Ubuntu that's running on UTC time.

According to someone on [Stackoverflow](https://stackoverflow.com/a/65069473/664933) this could be the cause.

Just verifying on my other machine (Mac) if my initial connection connection resolves this error as it's running on Local Time (Eastern/Toronto).

**Comment by paulywill** [comment]
Yup!
It was all about the time setting on my development machine! Wacky, but thankfully that Stackoverflow user pointed me in the right direction. Must be a wonderful gottchya-security-feature from MongoDB Atlas.

But once I confirmed that new cluster was working on my Mac, I tried the Ubuntu machine and it connected fine just like the previous setup database.

![image](https://user-images.githubusercontent.com/602422/120056593-c8822b00-c02c-11eb-8a8c-deb175a758e8.png)
